packages/react-variant-view/src/component/functionalPrediction/MutationAssessor.tsx

```diff
@@ -27,25 +27,32 @@ export default class MutationAssessor extends React.Component<
     IMutationAssessorProps,
     {}
 > {
-    private static MUTATION_ASSESSOR_URL: string =
-        'http://mutationassessor.org/r3/';
+    // Change to new url when manuscript available
```

Comment:
we should add a `TODO` here, right?
```suggestion
    // TODO Change to new url when manuscript available
```